src/yb/yql/pgwrapper/pg_wrapper.cc

```diff
@@ -349,14 +376,33 @@ Status PgWrapper::Start() {
   }
 
   argv.push_back("-c");
-  // TODO: we should probably load the metrics library in a different way once we let
-  // users change the shared_preload_libraries conf parameter.
+
+  // Gather the default extensions:
+  vector<string> metricsLibs;
+  metricsLibs.push_back("pg_stat_statements");
   if (FLAGS_pg_stat_statements_enabled) {
-    argv.push_back("shared_preload_libraries=pg_stat_statements,yb_pg_metrics,pgaudit,"
-      "pg_hint_plan");
-  } else {
-    argv.push_back("shared_preload_libraries=yb_pg_metrics,pgaudit,pg_hint_plan");
+    metricsLibs.push_back("yb_pg_metrics");
   }
+  metricsLibs.push_back("pgaudit");
+  metricsLibs.push_back("pg_hint_plan");
```

Comment:
The logic here seems changed. Previously `FLAGS_pg_stat_statements_enabled` determined if `pg_stat_statements` extensions is loaded. Now it toggles `yb_pg_metrics` instead.
I think this could be:
```
if (FLAGS_pg_stat_statements_enabled) {
  metricsLibs.push_back("pg_stat_statements");
}
metricsLibs.push_back("yb_pg_metrics");
metricsLibs.push_back("pgaudit");
metricsLibs.push_back("pg_hint_plan");
```

Reply:
Thank you for catching this. Addressed: https://github.com/yugabyte/yugabyte-db/pull/9576/commits/ab1d047dee772c55ffd9c4dba4f11470ec81f455.